extensions/amp-story/1.0/amp-story-interactive-poll.css

```diff
@@ -43,6 +44,17 @@
   flex: 100 !important;
   overflow: hidden !important;
   position: relative !important;
+  transition: flex-grow .5s var(--ease-out-curve) !important;
```

Comment:
Is there any way we could animate something else than the `flex-grow`? How hard would it be?
I'm not too worried about repaints as browsers should only repaint the flex container, but if there's an easy way to optimize this we should consider it.
cf https://csstriggers.com/flex-grow

Reply:
We would need to use transforms to make a considerable optimization.
I think we could use transformX on the typography and scale the background color but it would take some math and dom restructuring. That might be better in a separate CL.